File: modules/org.openhs.core.commons/src/org/openhs/core/commons/InfoStationData.java
package org.openhs.core.commons;

public class InfoStationData {
	
	public boolean validity = false;
	public String tmpInPath = "";
	public String tmpOutPath = "";
	
	public String toString() {
		
		String out = "";
				
		out = out + "\n validity: " + validity;
		out = out + "\n tmpInPath: " + tmpInPath;
		out = out + "\n tmpOutPath: " + tmpOutPath;
				
		return out;
	}	

}

File: web/InfoStation/src/app/OhsInfoStation/InfoStationData.ts

export class InfoStationData {

        public validity                 = false; // content of the forecast is valid
        public tmpInPath                = '';
        public tmpOutPath               = '';

        constructor() {

        }

        fillFromJSON(json: any) {

            for (var propName in json) {
                this[propName] = json[propName];
            }
        }
        
    }